### DISCUSSION TITLE
🇯🇵 List of resources in Japanese language

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/courses#37] [Ideas]
English is not the only language in the world. Many people do not speak English. Many creators do not make content in English. Therefore, we want to create separate lists of resources that can help people who want to build ML knowledge and do not speak that language. Indonesian is the next language we want to support.

I am looking for people who would like to help me. I don't know Japanese 😞 I need your help to make it happen.

**Comment by willianmoeda** [comment]
助けが必要です
